crates/rooch/src/commands/account/mod.rs

```diff
@@ -5,7 +5,8 @@ use crate::cli_types::CommandAction;
 use crate::commands::account::commands::balance::BalanceCommand;
 use async_trait::async_trait;
 use commands::{
-    create::CreateCommand, list::ListCommand, nullify::NullifyCommand, switch::SwitchCommand,
+    binding::BindingCommand, create::CreateCommand, list::ListCommand, nullify::NullifyCommand,
```

Comment:
Is there an option for updating the existing or created bitcoin address on Rooch?